### ISSUE TITLE
[Question] is the plugins same as ChatGPT official?

### ISSUE THREAD

**Issue by suwubee** [lobehub/lobe-chat#766]
### 🧐 问题描述 | Proposed Solution

I checked that the structures are the same, including openapi.yaml. If so, can it be transplanted in batches?
Maybe this is the power of Lobe?

**Comment by arvinxx** [comment]
Yes. We are compatible with ChatGPT pluign. https://x.com/lobehub/status/1736235125908885881?s=46&t=S01sh5U8Nv8bRLy2GXeJJw
